src/vector/platform/WebPlatform.ts

```diff
@@ -168,13 +168,7 @@ export default class WebPlatform extends VectorBasePlatform {
     };
 
     startUpdateCheck() {
-        super.startUpdateCheck();
-        this.pollForUpdate().then((updateState) => {
-            dis.dispatch<CheckUpdatesPayload>({
-                action: Action.CheckUpdates,
-                ...updateState,
-            });
-        });
+        //removed the code to disable version update
```

Comment:
Hi Samridhi, just like the other changes, please control this with a configuration variable (e.g. checkForUpdates) which defaults to true and we set to false in our config.json file